docs/tutorials/gcp_gpu.md

```diff
@@ -0,0 +1,195 @@
+*   [Creating a VM with attached GPUs](https://cloud.google.com/compute/docs/gpus/create-vm-with-gpus#create-new-gpu-vm)
+
+## 2. Prepare your computer
+
+Use SSH to create an encrypted tunnel from your computer to your VM and redirect
+a local port to your VM over the tunnel.
+
+1.  Follow the instructions in the
+    [Installing Cloud SDK](https://cloud.google.com/sdk/docs/install)
+    documentation to install the `gcloud` command line tool.
+2.  After installation, initialize the Google Cloud environment by using the
+    `gcloud init` command. You need to provide details about your VM, such as
+    the project name and the region where your VM is located.
+    1.  You can verify your environment by using the `gcloud config list`
+        command.
+3.  Create an SSH tunnel and redirect a local port to use the tunnel by typing
+    the following command in a terminal window on your computer. Replace
+    `[YOUR_INSTANCE_NAME]` with the name of your VM.
+
+```shell
+gcloud compute ssh [YOUR_INSTANCE_NAME]
+```
+
+When the command completes successfully, your prompt changes from your local
+machine to your virtual machine.
+
+## 3. Enable your virutal machine to use the GPu
+
+1.  Install the GPU driver. In the Google Cloud documentation, in the Installing
+    GPU drivers guide, follow the steps provided:
+    *   In
+        [the Examples section](https://cloud.google.com/compute/docs/gpus/install-drivers-gpu#examples),
+        under the Ubuntu tab. Only follow the steps for Ubuntu 20.04 (steps 3a
+        through 3f).
+    *   In the
+        [Verifying the GPU driver install](https://cloud.google.com/compute/docs/gpus/install-drivers-gpu#verify-driver-install)
+        section.
+2.  Run `sudo apt install -y nvidia-cuda-toolkit` to install the CUDA toolkit.
+3.  Add your CUDA toolkit to the environment search path.
+    1.  Run `ls /usr/local` to discover the directory of the CUDA toolkit that
+        you installed. The toolkit will be the highest number that looks like
+        the pattern `cuda-XX.Y`.  The output of the command should resemble the
+        following:
+
+        ```shell
+        bin cuda cuda-11 cuda-11.4 etc games include lib man sbin share src
+        ```
+
+        In this case, the directory is `cuda-11.4`.
+    2.  Add the CUDA toolkit path to your environment by appending the following
+        line to your `~/.bashrc` file.  Replace `[DIR]` with the CUDA directory that
+        you discovered in the previous step.
+
+        ```shell
+        echo "export PATH=/usr/local/[DIR]/bin${PATH:+:${PATH}}" >> ~/.bashrc
+        ```
+
+    3.  Run `source ~/.bashrc` to activate the new environment search path
+
+## 4. Install build tools
+
+Run the following command to install the Cmake tool and the pip tool. This step
+might take a few minutes to complete.
+
+```shell
+sudo apt install cmake && sudo apt install pip
```

Comment:
In my experience, users also need to install pybind11 after this step.
```
pip install pybind11
```

Reply:
Fixed.